src/app/components/main-page/main-page.component.html

```diff
@@ -20,12 +20,29 @@
       <a routerLink="/create-post" class="px-2 nav-link-style d-none d-sm-block">
         <fa-icon [icon]="faPlusSquare"></fa-icon>
       </a>
+
+      <div ngbDropdown class="d-inline-block">
+        <button class="img-button border-0 ml-auto" id="post-dropdown" ngbDropdownToggle>
```

Comment:
i added a bit of css to remove the dropdown arrow when i used a dropdown

Reply:
Done